### PR TITLE
chore(hybrid-cloud): Mark GitHub webhook endpoints as region only

### PR DESCRIPTION
**Pull request by dashed**

### CODE REVIEW

File: src/sentry/integrations/github/webhook.py [hunk 1]
```diff
@@ -517,6 +520,7 @@ def handle(self, request: Request) -> HttpResponse:
         return HttpResponse(status=204)
 
 
+@region_silo_endpoint
```
Comment (markstory) [inline src/sentry/integrations/github/webhook.py:523]: Some of GitHub's webhooks have to end up in control silo. The `installation.created` event is one that needs to stay in control.

File: src/sentry/integrations/github_enterprise/webhook.py [hunk 2]
```diff
@@ -168,6 +172,7 @@ def handle(self, request: Request) -> Response:
             return HttpResponse(status=204)
 
 
+@region_silo_endpoint
```
Comment (markstory) [inline src/sentry/integrations/github_enterprise/webhook.py:175]: Same here with the installation.created event.

File: src/sentry/integrations/github/webhook.py (hunk 1 above)
Comment (leeandher) [reply src/sentry/integrations/github/webhook.py:523]: We actually don't react to an `installation.created` webhook, only [the `installation.deleted` one](https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/github/webhook.py#L126) and in the parser it gets handled at the region silo with RPC methods.

**Review by leeandher — APPROVED** [review]
Thanks for this @dashed! I missed updating these types and decorators!

File: src/sentry/integrations/github/webhook.py (hunk 1 above)
Comment (dashed) [reply src/sentry/integrations/github/webhook.py:523]: At the control silo, we would intercept the Github webhook requests with this parser: https://github.com/getsentry/sentry/blob/01592b6a976ddafe103a0d693b01995565ac6976/src/sentry/middleware/integrations/parsers/github.py

Comment (markstory) [reply src/sentry/integrations/github/webhook.py:523]: My mistake. I conflated the github plugin and the integration. The plugin does use [installation.created webhook](https://github.com/getsentry/sentry/blob/31445be85fbd3ddd7c80dea12fa628795a52a6f6/src/sentry_plugins/github/webhooks/events/installation.py#L12-L21).

**Pull request merged**